WPFHospitalEditor/Controller/Interface/IMapObjectController.cs

```diff
@@ -8,6 +8,7 @@ interface IMapObjectController
         List<MapObject> getAllMapObjects();
         MapObject update(MapObject mapObject);
         List<MapObject> getOutterMapObjects();
-        MapObject findMapObjectById(int id);
+        MapObject findMapObjectById(int id, List<MapObject> mapObjects);
```

Comment:
You call this method in like 10 places (controller, service, repo) and only in tests you don't send list of all map objects. So you basically send list of all objects many times.  You should get list of all objects in repo, and for tests you need to mock response from repo.

Reply:
It has only id as parameter now.